fe-ionic/Jenkinsfile.template

```diff
@@ -22,7 +22,7 @@ odsComponentPipeline(
 def stageBuild(def context) {
   stage('Build') {
     withEnv(["TAGVERSION=${context.tagversion}", "NEXUS_HOST=${context.nexusHost}"]) {
-      sh "npm i -g @ionic/cli@6.3.0"
+      sh "npm i -g @ionic@5.4.16"
```

Comment:
The command as it is doesn't work on my machine. Isn't this missing something? E.g. ``npm i -g @ionic/core@5.4.16``?

Reply:
Added new pull request for this one, one too many @